main/store/actions/index.js

```diff
@@ -134,12 +136,18 @@ module.exports = {
       return updated
     })
   },
-  updateAccount: (u, updatedAccount) => {
-    u('main.accounts', updatedAccount.id, account => {
-      // if (account) return updatedAccount // Account exists
-      // if (add) return updatedAccount // Account is new and should be added
-      return { ...updatedAccount, balances: (account || {}).balances }
+  updateAccount: (u, updatedAccount, newName) => {
+    const { id } = updatedAccount
+    u('main.accounts', id, (account = {}) => {
+      return { ...updatedAccount, balances: account.balances || {} }
     })
+    const name = newName || updatedAccount.name
```

Comment:
do we need the newName parameter? isnt it always the same as `updatedAccount.name`?

Reply:
You are correct